packages/components/src/local/molecules/option-menu/index.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import { faBath, faClone, faFileDownload, faTrash } from '@fortawesome/free-solid-svg-icons'
+
+// Import component files
+import OptionMenu from './index'
+import docs from './README.md'
+
+export default {
+  title: 'local/molecules/OptionMenu',
+  component: OptionMenu,
+  decorators: [],
+  parameters: {
+    readme: {
+      // Show readme before story
+      content: docs
+    }
+  }
+}
+
+export const Default: React.FC = () => {
+  const menus = [
+    {
+      onClick: (): void => window.alert('Click on menu #1'),
+      icon: faBath,
+      title: 'Save clean copy of wargame (no messages)',
+      label: 'Clean copy'
+    },
+    {
+      onClick: (): void => window.alert('Click on menu #2'),
```

Comment:
The `menus` constant is duplicated in the spec file. It would be better to move this to an external file and reference it in both. That way we are not repeating code.

Reply:
Added mocks for menus